share/getters/shrex_test.go

```diff
@@ -166,8 +200,8 @@ func newStore(t *testing.T) (*eds.Store, error) {
 func generateTestEDS(t *testing.T) (*rsmt2d.ExtendedDataSquare, da.DataAvailabilityHeader, nmtnamespace.ID) {
 	eds := share.RandEDS(t, 4)
 	dah := da.NewDataAvailabilityHeader(eds)
-	randNID := dah.RowRoots[(len(dah.RowRoots)-1)/2][:namespace.NamespaceSize]
-	return eds, dah, randNID
+	max := nmt.MaxNamespace(dah.RowRoots[(len(dah.RowRoots))/2-1], ipld.NamespaceSize)
```

Comment:
did you mean to do `dah.RowRoots[(len(dah.RowRoots))/2-1` ?

Reply:
yep, sneaky fix to always select last row in ods. Previous code relied on int division rounding, that I think could be confusing.